idb/postgres/internal/encoding/decode.go

```diff
@@ -0,0 +1,8 @@
+package encoding
```

Comment:
This could be convenient as a top-level package to replace all of the other places we use `github.com/algorand/go-algorand-sdk/encoding` and `encoding/json`.

Reply:
I sort of intended to make it exclusive to `idb/postgres/` so that if we wanted to change the json format for logging, etc, we could do that easily without affecting the DB layer.